### PR TITLE
spicedao.eth update SKIN & Avatar

### PR DESCRIPTION
**Pull request by bitsandtea**

### CODE REVIEW

File: skins/spicedao.scss [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.spicedao {
+    --primary-color: #000;
+    --bg-color: #1201ff;
+    --link-color: white;
+    --heading-color: #000;
+    --border-color: #000;
+    --header-bg: #cf0;
+    --block-bg: transparent;
+    --text-color: #fff
```
Comment (gokaiorg) [inline skins/spicedao.scss:9]: You forgot to close this line.
```suggestion
    --text-color: #fff;
```

**Review by gokaiorg — CHANGES_REQUESTED** [review]

**Comment by bitsandtea** [comment]
Updated it @gokaiorg

**Review by gokaiorg — APPROVED** [review]

**Comment by gokaiorg** [comment]
There is a conflict in your PR, can you resolve it?

**Comment by bitsandtea** [comment]
@gokaiorg I am not sure how to do that .. I have pushed two image files and the scss file. Any ideas?

**Comment by gokaiorg** [comment]
You could try update your fork with something like this
`git remote add master https://github.com/snapshot-labs/snapshot-spaces.git
git fetch master
git checkout master
git merge master/master`
Or fork again and push your update.

**Comment by bitsandtea** [comment]
> You could try update your fork with something like this
> `git remote add master https://github.com/snapshot-labs/snapshot-spaces.git`
> `git fetch master`
> `git checkout master`
> `git merge master/master`
> Or fork again and push your update.

okay, looks like there are no conflicts anymore Thanks for the hint @gokaiorg

**Pull request merged**